tools/ci_build/github/azure-pipelines/c-api-noopenmp-packaging-pipelines.yml

```diff
@@ -357,7 +453,7 @@ jobs:
   - task: BatchScript@1
     displayName: 'Bundle Native NuGet and other binaries'
     inputs:
-      filename: $(Build.SourcesDirectory)\tools\ci_build\github\windows\bundle_dlls_gpu.bat
+      filename: $(Build.SourcesDirectory)\tools\ci_build\github\windows\bundle_dlls_gpu_nuget.bat
```

Comment:
Why this line needs be changed?

Reply:
This bat file is mainly for bundling libraries for nuget package, simply rename it for clarity.
Also, we need to bundle libraries for GPU zip file, so the name "bundle_dlls_gpu.bat" is better for this case.